lib/plugins/expressionParser.js

```diff
@@ -28,8 +28,9 @@
 const Parser = require('jison').Parser;
```

Comment:
CNR entry should be added (do we have a related issue?). Seems to be an important fix :)

What about the IOTA code? If we have copied the same pattern, the bug would also exists there...

Reply:
done in https://github.com/telefonicaid/iotagent-node-lib/pull/1192/commits/f65e2b4e79e09c772c31e31620a40618172fa6ef